process_request/tests.py

```diff
@@ -280,8 +302,8 @@ def test_send_aeon_requests(self, mock_get_data):
         self.assertTrue(isinstance(delivered, dict))
 
         request_type = "foo"
-        with self.assertRaises(Exception, msg="Unknown request type '{}', expected either 'readingroom' or 'duplication'".format(request_type)):
-            AeonRequester().get_request_data(request_type, **data)
+        with self.assertRaises(ValueError, msg="Unknown request type '{}', expected either 'readingroom' or 'duplication'".format(request_type)):
+            AeonRequester().get_request_data(request_type, "https://dimes.rockarch.org", **data)
```

Comment:
This is great - is there a chance we can use the same kind of detail in assertRaises for other exceptions where we're testing the message (like line 290 above)?

Reply:
I _think_ I've gotten what you were asking, but let me know if I missed the mark.